src/Domain/LeanCode.CQRS.RemoteHttp.Client/HttpQueriesExecutor.cs

```diff
@@ -23,7 +23,7 @@ public HttpQueriesExecutor(HttpClient client)
             this.serializerOptions = serializerOptions;
         }
 
-        public virtual async Task<TResult> GetAsync<TResult>(IRemoteQuery<TResult> query)
+        public virtual async Task<TResult?> GetAsync<TResult>(IRemoteQuery<TResult> query)
```

Comment:
That's not correct also. This should never return `null`. The response is malformed is null is returned here.

Reply:
I mean, technically it could, but only if `TResult` itself is a nullable type.

Reply:
Ah, yeah, indeed. That's the only case.